client/src/components/molecules/style/RequestsTable.scss

```diff
@@ -0,0 +1,42 @@
+.request-table {
+    @include table-base;
+    h1 {
+        @include section-header-text;
+        padding-bottom: 4px;
+        margin: 0;
+    }
+    width:1200px;
+    margin-left: auto;
+    margin-right:auto;
+    .edit{
+        color: #333333;
+    }
+    .delete{
+        color: #333333;
+    }
+    & td {
+      border-top: unset;
+      padding: 0.6rem;
+      vertical-align: unset;
+    }
+    .row-style{
```

Comment:
Just use tbody tr instead of creating a new class. Also font-size: 16px is already in @mixin table-base, so remove (and all other places where style is unnecessarily duplicated)

Reply:
Got it. thanks